index.html

```diff
@@ -594,7 +600,7 @@ <h2>Verification Methods</h2>
                 <dt>controller</dt>
                 <dd>
 The value of the <code>controller</code> property MUST be a <a
-data-cite="INFRA#string">string</a> that conforms to the [[URL]] syntax.
+data-cite="INFRA#string">string</a> that conforms to the URI syntax as defined in [[RFC3986]].
```

Comment:
path, query and fragment allowed? or not?

Reply:
This deviates from the use of URL in DID Core -- https://www.w3.org/TR/did-core/#verification-methods

-1 for deviating from DID Core unless we have a very good reason to do so.

It was expected that a controller could include a path, query, and fragment. 

We don't use URIs here because it's not defined how to retrieve the resources, whereas a URL does tell you how to retrieve the resource. Data Integrity requires you to fetch these resources, so any usage of a URI would also need to define how to get the document at that resource. 

For example, `controller` documents MUST be retrieved to see if there is a bidirectional relationship between the cryptographic material and the controller. If you don't do this, you're susceptible to all sorts of terrible attacks that other cryptographic systems that do not do this bi-directional checking are susceptible to.